[2022-12-08 11:27:58] local.ERROR: Uncaught Symfony\Component\Finder\Exception\DirectoryNotFoundException: The "C:\xampp\htdocs\ahmad\property_portal\resources\views" directory does not exist. in C:\xampp\htdocs\ahmad\vendor\symfony\finder\Finder.php:638
Stack trace:
#0 C:\xampp\htdocs\ahmad\vendor\laravel\framework\src\Illuminate\Filesystem\Filesystem.php(564): Symfony\Component\Finder\Finder->in('C:\\xampp\\htdocs...')
#1 Command line code(1): Illuminate\Filesystem\Filesystem->files('C:\\xampp\\htdocs...')
#2 Command line code(1): getViews('C:\\xampp\\htdocs...', Object(Illuminate\Filesystem\Filesystem))
#3 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Symfony\\Component\\Finder\\Exception\\DirectoryNotFoundException: The \"C:\\xampp\\htdocs\\ahmad\\property_portal\\resources\\views\" directory does not exist. in C:\\xampp\\htdocs\\ahmad\\vendor\\symfony\\finder\\Finder.php:638
Stack trace:
#0 C:\\xampp\\htdocs\\ahmad\\vendor\\laravel\\framework\\src\\Illuminate\\Filesystem\\Filesystem.php(564): Symfony\\Component\\Finder\\Finder->in('C:\\\\xampp\\\\htdocs...')
#1 Command line code(1): Illuminate\\Filesystem\\Filesystem->files('C:\\\\xampp\\\\htdocs...')
#2 Command line code(1): getViews('C:\\\\xampp\\\\htdocs...', Object(Illuminate\\Filesystem\\Filesystem))
#3 {main}
  thrown at C:\\xampp\\htdocs\\ahmad\\vendor\\symfony\\finder\\Finder.php:638)
[stacktrace]
#0 {main}
"} 
